src/App.css

```diff
@@ -88,3 +88,9 @@ body {
   font-weight: bold !important;
   color: #666666 !important;
 }
+
+.syntax-highlight {
+  color: greenyellow;
```

Comment:
```suggestion
  color: #89c999;
```

Reply:
how's this?

Reply:
I am color blind so it all looks grey to me